python/scannerpy/stdlib/poses.py

```diff
@@ -42,6 +43,7 @@ class Pose(object):
                    [255, 0, 170], [170, 0, 255], [255, 0, 255], [85, 0, 255]]
 
     def __init__(self):
+        self.posescores = 0
```

Comment:
Can you rename this to `pose_score`? Pose and score are two separate words and it should be singular since this only represents one pose's score.